exchanges/okx/okx_websocket.go

```diff
@@ -1044,20 +1039,47 @@ func (ok *Okx) wsProcessOrders(respRaw []byte) error {
 		if err != nil {
 			return err
 		}
-		ok.Websocket.DataHandler <- &order.Detail{
-			Price:           response.Data[x].Price,
-			Amount:          response.Data[x].Size,
-			ExecutedAmount:  response.Data[x].LastFilledSize.Float64(),
-			RemainingAmount: response.Data[x].AccumulatedFillSize.Float64() - response.Data[x].LastFilledSize.Float64(),
-			Exchange:        ok.Name,
-			OrderID:         response.Data[x].OrderID,
-			Type:            orderType,
-			Side:            response.Data[x].Side,
-			Status:          orderStatus,
-			AssetType:       a,
-			Date:            response.Data[x].CreationTime,
-			Pair:            pair,
+		var amount, quoteAmount, remAmount float64
+
+		if response.Data[x].SizeType == "quote_ccy" {
+			quoteAmount = response.Data[x].Size.Float64()
+		} else {
+			amount = response.Data[x].Size.Float64()
+		}
+
+		execAmount := response.Data[x].AccumulatedFillSize.Float64()
+
+		// We can only work out the
```

Comment:
Is this supposed to run onto `amount` below or is this comment incomplete?

Reply:
Fixed during rebase.
Thanks for spotting